src/toxcore/messenger/conference/packet/action.rs

```diff
@@ -0,0 +1,93 @@
+/*! Conference action action struct.
+*/
+
+use std::str;
+use nom::{be_u16, be_u32, rest};
+
+use crate::toxcore::binary_io::*;
+
+/** Action is the struct that holds info to send action to a conference.
+
+Sent to notify action to all member of conference.
+
+Serialized form:
+
+Length    | Content
+--------- | ------
+`1`       | `0x63`
+`2`       | `conference number`
+`2`       | `peer number`
+`4`       | `action number`
+`1`       | `0x41`
+variable  | `action`(UTF-8 C String)
+
+*/
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct Action {
+    conference_number: u16,
```

Comment:
Shouldn't these fields be public? Otherwise how can we process group packets?

Reply:
Evgeny, should't conference_number be conference_id? And same for other fields?

Reply:
Yep, it seems they are ids.